Update Index.md to add clarification tools usage (#13015)
dotnet/docs: docs/core/porting/index.md

The following list shows tools you might find helpful to use during the porting process:

> * .NET Portability Analyzer - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer), a tool that can generate a report of how portable your code is between .NET Framework and your target .NET Core platform. The report contains an assembly-by-assembly breakdown of the Type and APIs missing on the target .NET Core platform . For more information, see [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md). You may want to run .NET Portability Analyzer before your porting starts to evaluate the missing APIs gap  

Reviewer: ```suggestion
* .NET Portability Analyzer - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=ConnieYau.NETPortabilityAnalyzer), a tool that can generate a report of how portable your code is between .NET Framework and your target .NET Core platform. The report contains an assembly-by-assembly breakdown of the Type and APIs missing on the target .NET Core platform. For more information, see [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md). It is recommended to run the .NET Portability Analyzer tool before you start porting, as it will help you identify any gaps in missing APIs.
```

* .NET API analyzer - A Roslyn analyzer that discovers potential compatibility risks for C# APIs on different platforms and detects calls to deprecated APIs. For more information, see [.NET API analyzer](../../standard/analyzers/api-analyzer.md).  This analyzer is helpful after you already created your .NET Core project to identify runtime behavior differences on different platforms. Currently, the .NET API Analyzer reports the following cases.

> 	-   Usage of a .NET Standard API that throws [PlatformNotSupportedException](https://docs.microsoft.com/en-us/dotnet/api/system.platformnotsupportedexception) (PC001).

Reviewer: ```suggestion
	-   Use of a .NET Standard API that throws [PlatformNotSupportedException](https://docs.microsoft.com/en-us/dotnet/api/system.platformnotsupportedexception) (PC001).
```

> 	-   Use of a .NET Standard API that throws [PlatformNotSupportedException](https://docs.microsoft.com/en-us/dotnet/api/system.platformnotsupportedexception) (PC001).

Reviewer 2: These use cases are all covered in the other topic already: https://docs.microsoft.com/en-us/dotnet/standard/analyzers/api-analyzer#supported-diagnostics

Do we really need to call it out here again? If so, to keep both in sync, I'd move the text to an [include file](https://review.docs.microsoft.com/en-us/help/contribute/includes-best-practices?branch=master) and reuse that file in both places. The existing text has the appropriate Markdown for this including how we format API links. If you want to use an API link for PC004, just make sure you use the right casing for the API name since it's case-sensitive.

You can see an example of this at https://github.com/dotnet/docs/blob/master/docs/standard/net-standard.md - look for !INCLUDE.

> 	-   Usage of a .NET Standard API that isn't available on the .NET Framework 4.6.1 (PC002).

Reviewer: ```suggestion
	-   Use of a .NET Standard API that isn't available on the .NET Framework 4.6.1 (PC002).
```

> 	-   Usage of a native API that doesn’t exist in UWP (PC003).

Reviewer: ```suggestion
	-   Use of a native API that doesn’t exist in UWP (PC003).
```

> 	-   Usage of Delegate.BeginInvoke and EndInvoke APIs (PC004).

Reviewer: ```suggestion
	-   Use of <xref:system.windows.forms.control.begininvoke> and <xref:system.windows.forms.control.endinvoke> APIs (PC004).
```

> 	-   Usage of an API that is marked as deprecated (DEXXXX).

Reviewer: ```suggestion
	-   Use of an API that is marked as deprecated (DEXXXX).
```
